src/cmd/devdraw/mkwsysrules.sh

```diff
@@ -60,7 +60,14 @@ elif [ $WSYSTYPE = osx ]; then
 	echo 'WSYSOFILES=$WSYSOFILES osx-screen-carbon-objc.o osx-draw.o osx-srv.o'
 	echo 'MACARGV=macargv.o'
 elif [ $WSYSTYPE = osx-cocoa ]; then
-	echo 'WSYSOFILES=$WSYSOFILES osx-draw.o cocoa-screen-objc.o cocoa-srv.o cocoa-thread.o'
+	if sw_vers | grep 'ProductVersion:	10\.14' >/dev/null; then
```

Comment:
Would it make sense to set this up so it will keep use this renderer in 10.15 and future versions too?

E.g. we could make the matcher for the old cocoa renderer explicitly match up to 10.13 and make the new one the fallback case.